dist/ci/travis_script.sh

```diff
@@ -33,6 +33,7 @@ if test -z "$SUBTEST"; then
       bundle exec rails test:spider
       ;;
     linter)
+      bundle exec rake db:structure:verify
```

Comment:
That's what we agreed on in our standup. Where would you put it? And does it really matter?